### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: .ci/settings/settings.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
 CONTENT_ORIGIN = "http://localhost:8080/"
 ALLOWED_EXPORT_PATHS = ["/tmp"]
-ORPHAN_PROTECTION_TIME = 0
```
Comment (lubosmj) [inline .ci/settings/settings.py:3]: What led us to this removal?

File: pulpcore/cli/ostree/distribution.py [hunk 2]
```diff
@@ -74,11 +74,13 @@ def distribution(ctx: click.Context, pulp_ctx: PulpCLIContext, distribution_type
 @pass_pulp_context
 def update(
     pulp_ctx: PulpCLIContext,
-    distribution_ctx: PulpOstreeDistributionContext,
+    distribution_ctx: PulpEntityContext,
     base_path: Optional[str],
     repository: Optional[Union[str, PulpEntityContext]],
     version: Optional[int],
 ) -> None:
+    assert isinstance(distribution_ctx, PulpOstreeDistributionContext)
```
Comment (lubosmj) [inline pulpcore/cli/ostree/distribution.py:82]: Was this necessary? I am not a fan of writing asserts because of type change.

**Review by lubosmj — APPROVED** [review]

File: .ci/settings/settings.py (hunk 1 above)
Comment (lubosmj) [reply .ci/settings/settings.py:3]: Is the settings file even used?

Comment (mdellweg) [reply .ci/settings/settings.py:3]: It's not needed. And running more with default settings is better, IMHO.

File: pulpcore/cli/ostree/distribution.py (hunk 2 above)
Comment (mdellweg) [reply pulpcore/cli/ostree/distribution.py:82]: Yes, this is necessary to hint that the passed context object is really of that type.

Comment (lubosmj) [reply pulpcore/cli/ostree/distribution.py:82]: I meant the whole change, with PulpEntityContext.

Comment (mdellweg) [reply pulpcore/cli/ostree/distribution.py:82]: I know. but the typing hints of click improved and so that is needed too.

**Pull request merged**